src/Sarif.Multitool/Rules/UriBaseIdRequiresRelativeUri.cs

```diff
@@ -29,47 +33,14 @@ protected override IEnumerable<string> FormatIds
             }
         }
 
-        protected override void Analyze(
```

Comment:
>Analyze [](start = 32, length = 7)

There used to be several objects with pairs of properties `uri`, `uriBaseId`. Now, only the `fileLocation` object has this pair, so we only have to check for this condition in one place. We do have to make sure to visit all the `fileLocation` objects in the model.